docs/src/toml.md

```diff
@@ -35,6 +36,19 @@ type = "float"
 description = "Molecular weight dry air (kg/mol)"
 ```
 
+### Properly tagged parameter
```

Comment:
One should highlight in the comments that the search for tagging is done via this "fuzzy" matching, 

It may be useful to export fuzzy_match so users can test how their tags are interpreted in the current fuzzy_match function
e.g. currently it may not be clear to users that
```julia
fuzzy_match("Grachev", "gR*__aC.h,._e, --v-") # returns true
fuzzy_match("Grachev", "&gR*__aC.h,._e, --v-") # returns false
```